spec/factories/users.rb

```diff
@@ -32,6 +32,7 @@
 
     after(:create) do |user|
       user.is_admin = false
+      user.is_disabled = false
       # save with bang cause we want change in DB and not just in object instance
       user.save!
```

Comment:
Ok, maybe I misread the comment. It's saying that we want to persist the change in attribute. The fact that it's "with bang" is irrelevant.

Still raises the question of why we don't set these attributes like normal in a factory (ie (`is_disabled true`)